scripts/AbinsModules/LoadCRYSTAL.py

```diff
@@ -120,18 +123,18 @@ def _determine_dispersion(self):
 
         return phonon_dispersion
 
-    def _read_lattice_vectors(self, obj_file=None):
+    def _read_lattice_vectors(self, file_obj=None):
         """
         Reads lattice vectors from .out CRYSTAL file.
-        :param obj_file:  file object from which we read
+        :param file_obj:  file object from which we read
         :return: list with lattice vectors
         """
-        self._find(file_obj=obj_file, msg="DIRECT LATTICE VECTORS CARTESIAN COMPONENTS (ANGSTROM)")
-        obj_file.readline()  # Line: X                    Y                    Z
+        self._parser.find_first(file_obj=file_obj, msg="DIRECT LATTICE VECTORS CARTESIAN COMPONENTS (ANGSTROM)")
+        file_obj.readline()  # Line: X                    Y                    Z
         dim = 3
         vectors = []
         for i in range(dim):
-            line = obj_file.readline()
+            line = file_obj.readline()
             line = line.split()
             vector = []
```

Comment:
A small point here - could vector be initialised with a list comprehension here `vector = [float(item) for item in line]`. This would be faster and more pythonic